TransferMethod/Sources/View/Widget/TextWidget.swift

```diff
@@ -258,12 +258,36 @@ class TextWidget: AbstractWidget {
             range: nil)
     }
 
-    private func getUnformattedText() -> String {
+    func getUnformattedText() -> String {
```

Comment:
why is private removed?

Reply:
because we are overriding it in `ExpiryDateWidget`